frontend/src/app/person-detail-dialog/person-detail-dialog.component.ts

```
@@ -0,0 +1,68 @@
+  }
+
+  forceDownload(imageUrl: string) {
+    fetch(imageUrl)
+      .then(response => {
+        // Get the filename from the Content-Disposition header if available
+        const contentDisposition = response.headers.get('Content-Disposition');
+        let filename = 'download.jpg'; // Default filename
+
+        if (contentDisposition) {
+          const match = contentDisposition.match(/filename="?([^"]+)"?/);
+          if (match && match[1]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)